### PR TITLE
updated the installation guide. 

### PR DESCRIPTION
**Pull request by nnenasheva**
Added steps. Made it easier to read.Removed an outdated directive (won't work)

### CODE REVIEW

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for about an hour for the submission to be processed.

**Comment by artml** [comment]
1. We never mention the supported browsers and, IMO, we should. The best place is at the very beginning, along with other system requirements, and then refer this part again in the section 5a "Start the wizard".

File: book/installation.rst [hunk 1]
```diff
@@ -6,210 +6,206 @@
...
+        $ cd [$folder_location]
         $ git clone -b 1.9.0 https://github.com/orocrm/platform-application.git orocrm
 
-Next, you'll need to install the necessary dependencies which may take some time:
+**1a.3.** Go to the "orocrm" directory and run the *composer install* command, in order to install the dependencies. 
 
 .. code-block:: bash
 
+    $ cd orocrm
     $ composer install
```
Comment (rgrebenchuk) [inline book/installation.rst]: Recommend to use composer install --prefer-dist --no-dev . In other case user get error or all dev environment will be installed

File: book/installation.rst [hunk 2]
```diff
@@ -6,210 +6,206 @@
...
+    websocket_frontend_host ('*'):
+    websocket_frontend_port (8080):
+    websocket_backend_host ('*'):
+    websocket_backend_port (8080):
     session_handler (session.handler.native_file):
     locale (en):
     secret (ThisTokenIsNotSoSecretChangeIt):
+    installed (null):
+    assets_version (null):
+
```
Comment (rgrebenchuk) [inline book/installation.rst]: There is new parameter assets_version_strategy
    assets_version_strategy: time_hash # A strategy should be used to generate the global assets version, can be:
                                       # null        - the assets version stays unchanged
                                       # time_hash   - a hash of the current time
                                       # incremental - the next assets version is the previous version is incremented by one (e.g. 'ver1' -> 'ver2' or '1' -> '2')

File: book/installation.rst [hunk 3]
```diff
@@ -6,210 +6,206 @@
...
 .. code-block:: bash
 
-    $ cd /var/www/vhosts
+    $ cd [$folder_location]
     $ wget -c http://www.orocrm.com/downloads/crm-application.tar.gz
     $ tar -xzvf crm-application.tar.gz
 
-The source code archive already ships with the libraries installed in its ``vendor`` directory. You
-should now run Composer to update them to their latest supported versions:
+**1b.2.** Run the *composer update* command to update the downloaded libraries to the latest supported versions.
```
Comment (rgrebenchuk) [inline book/installation.rst]: *composer update* -> composer install --prefer-dist --no-dev

File: book/installation.rst [hunk 4]
```diff
@@ -256,175 +252,219 @@ If you are using **Nginx** as web server your virtual host configuration should
...
+5b. Using the Installation Command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use the ``oro:install`` console command can to trigger the installation
-from the command line:
+Another way to run the installation script is with the *oro:install* command in the console. 
 
 .. code-block:: bash
 
     $ php app/console oro:install
```
Comment (rgrebenchuk) [inline book/installation.rst]: add --env prod for production mode. In other case dev environment will be used

**Pull request merged**